### PR TITLE
created CallToAction component

### LINKED ISSUE #35: Create CallToAction section.
### Describe the Current Behavior/Feature:

Create call to action section.

[Partner call to action](https://www.figma.com/file/QWycOkxNJmtiyZ3v8imIzx/Code-PDX-website?type=design&node-id=0-865&mode=design&t=X85qMIPZEmkx8zws-4)

---

### Proposed Implementation:

Screenshot 2023-10-29 at 4 01 02 PM

### PR DESCRIPTION
**Pull request by Jared-Krajewski**
## This PR:

Resolves #35 

---

## Screenshots (if applicable):

Screenshot 2023-10-29 at 5 06 46 PM

### CODE REVIEW

**Review by andycwilliams — APPROVED** [review]
Looks good. Just a couple of thoughts.

Just a couple of things I'd like to try:
- Place the Contact Us button a bit closer to the text. (For the laptop viewport. With mobile it looks fine).
- Reduce the margins between this and the section above it. (Right now it's the Volunteer section but will eventually be the Partner section). Maybe not by a whole lot. Just trim the whitespace a little.

Capture

My only real comments are suggestions with formatting, but they're pretty subjective so I'll approve this now.

**Comment by Jared-Krajewski** [comment]
@andycwilliams I added a % margin to the right of button and decreased padding at top. some of this may need to be re checked once all of the components are there.

File: src/components/home/partners/CallToAction.jsx [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
+          <Typography variant="h4" pb={'2rem'}>
+            Empower Change: Become a partner in our civic tech movement.
+          </Typography>
+          <Typography variant="h4">
+            Have an idea for a civic project in Portland or want to partner with us?
+          </Typography>
+        </Grid>
+        <Grid item xs={12} md={4} style={{ textAlign: 'center', alignItems: 'center' }}>
+          <Button
+            sx={{
```
Comment (xscottxbrownx) [inline src/components/home/partners/CallToAction.jsx:20]: This isn't doing anything, needs to be on the `Grid item`

add `mt={{ xs: '50px', md: '0' }}` to line 18 to see.

**Review by xscottxbrownx — CHANGES_REQUESTED** [review]
The only other thing is the space around the button/size of button.

I think I'd prefer the button from the hero section here, and this one in the hero section.  Just a thought.

**Review by xscottxbrownx — APPROVED** [review]

**Pull request merged**